packages/remark-stringify/lib/visitors/table-cell.js

```diff
@@ -2,6 +2,10 @@
 
 module.exports = tableCell
 
+var lineFeed = /\n/g
```

Comment:
Are carriage returns `\r` also a concern?

Reply:
```suggestion
var lineFeed = /\r?\n/g
```

Right! Should be there as well.
`remark-parse` never puts first removes those, but also doesn’t put newlines in, so this edge case only happens if someone somehow injects them, and they could just as well inject carriage returns.